Frontend/ui-customer/src/Strings/EN.js

```diff
@@ -215,7 +216,7 @@ const englishStrings = {
   "BOOK_RIDE_": "Book Ride",
   "CANCEL_AUTO_ASSIGNING": "Cancel Auto Assigning",
   "LOCATION_UNSERVICEABLE": "Location unserviceable",
-  "CURRENTLY_WE_ARE_LIVE_IN_": "We are not live in your area yet!",
+  "CURRENTLY_WE_ARE_LIVE_IN_": "We are not live in your area yet!\n\n You can access ride history and other settings from the menu",
```

Comment:
This text is not updated in other files, KA, ML, BA, FR?

Reply:
done